src/aws.cr

```diff
@@ -1,12 +1,18 @@
 module AWS
-  VERSION = "0.1.0"
+  VERSION = "0.1.1"
 
   class_property access_key_id : String = ENV["AWS_ACCESS_KEY_ID"]? || ""
   class_property secret_access_key : String = ENV["AWS_SECRET_ACCESS_KEY"]? || ""
   class_property region : String = ENV["AWS_REGION"]? || ""
+  class_property use_tls : Bool = case ENV["AWS_USE_TLS"]?
```

Comment:
Something like `ENV["AWS_USE_TLS"] == "false" ? false : true` achieve the same logic

Reply:
True, I didn't know you can do that in Crystal